### PR TITLE
Combined Home Page and Ticket Search Pages and wrote Unit Tests

### PR DESCRIPTION
**Pull request by ctwoolsey**

### CODE REVIEW

**Comment by abobwhite** [comment]
@ctwoolsey Please merge the latest develop into this branch since it is now in conflict

**Comment by ctwoolsey** [comment]
Do I do this by doing a pull, while this branch is selected and open in
SourceTree?  Or is this something manual I have to do to the code itself?

**Comment by abobwhite** [comment]
Go to develop locally, pull. Then switch to this branch locally, and merge develop into the branch. Resolve any conflicts. Then Commit and Push this branch. The PR will be updated automatically.

**Comment by ctwoolsey** [comment]
OK, I think I merged the files correctly.

File: app/styles/main.scss [hunk 1]
```diff
@@ -154,13 +182,13 @@ $homepageTileHeight : 250px;
...
-.homepageTile:hover {
+.homepageTile:hover, .court-search-bar:hover {
   border-color: $blue;
   cursor: pointer;
 }
 
-.homepageTile.unselected {
-  opacity: 0.5;
+.homepageTile.unselected, .court-search-bar.unselected {
+  opacity: 0.7;
```
Comment (abobwhite) [inline app/styles/main.scss]: You just didn't like the opacity so you decided to change it? I'm sure it's not a big difference but the point was to keep eyes focused on the selection and to really make it pop. We can look at this further when we polish everything.

**Comment by ctwoolsey** [comment]
Yes, the way the picture comes through was distracting so I thought.  I
just upped it a little.  Do you want me to undo this or keep with changes
and then we can look at this more later?

**Comment by abobwhite** [comment]
I merged it. It's fine for now. We can take a look at it later. We'll probably want to meet in person when we're closer to launch to go over the UI/UX of everything and we can decide if this needs to change or not.

**Pull request merged**